universal-login-sdk/lib/core/observers/BalanceObserver.ts

```diff
@@ -1,49 +1,49 @@
-import {providers} from 'ethers';
-import {SupportedToken, ensure, RequiredBalanceChecker, BalanceChecker} from '@universal-login/commons';
-import {ConcurrentDeployment} from '../utils/errors';
 import ObserverRunner from './ObserverRunner';
-
-export type ReadyToDeployCallback = (tokenAddress: string, contractAddress: string) => void;
+import {BalanceChecker, TokenDetails, TokenDetailsWithBalance} from '@universal-login/commons';
+import deepEqual = require('deep-equal');
 
 export class BalanceObserver extends ObserverRunner {
```

Comment:
#NextPR To consider
Rename `ObserverBase` -> `EventObserverBase`
Introduce new `ObserverBase` with subscribe and unsubscribe?